src/main/java/net/masterthought/cucumber/ReportParser.java

```diff
@@ -97,4 +94,57 @@ private void setMetadata(Feature[] features, String jsonFile, int jsonFileNo) {
             feature.setMetaData(jsonFile, jsonFileNo, configuration);
         }
     }
+
+    /**
+     * Parses passed properties files and adds metadata to overview-features page
+     *
+     * @param propertiesFiles
+     *            property files to read
+     */
+    public void parsePropertiesFiles(List<String> propertiesFiles) {
+
+        if (propertiesFiles.isEmpty()) {
+
+            return;
+
+        }
+
+        for (String propertyFile : propertiesFiles) {
+
+            LOG.info("Processing MetadataFile - %s", propertyFile);
+
+            processMetaDataFile(propertyFile);
+
+        }
+
+    }
+
+    private void processMetaDataFile(String file){
```

Comment:
metadata or classification?

Reply:
We are processing classifications within a metaDataFile...maybe processClassificationsWithinMetaDataFile if that is good with you?

Reply:
I thought about `processClassificationFile` - what do you think?

Reply:
sounds good will update to that